core/trino-main/src/main/java/io/trino/sql/planner/OptimizerConfig.java

```diff
@@ -474,16 +474,16 @@ public OptimizerConfig setOptimizeMetadataQueries(boolean optimizeMetadataQuerie
     }
 
     @Nullable
-    public MarkDistinctStrategy getMarkDistinctStrategy()
```

Comment:
It's confusing to see this config getting deleted here and then added back again in another commit. Can we avoid that ?